test-cases/performance/perf-scylla-cassandra-regular.yaml

```diff
@@ -0,0 +1,51 @@
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=625000001..703125000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=703125001..781250000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=781250001..859375000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=859375001..937500000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=937500001..1015625000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1015625001..1093750000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1093750001..1171875000",
+                    "cassandra-stress write no-warmup cl=ALL n=78125000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1171875001..1250000000"
+                ]
+
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=1250000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..1250000000"
+stress_cmd_r: "cassandra-stress read no-warmup  cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=90' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=gauss(1..1250000000,625000000,1250000000)' "
+stress_cmd_m: ["cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=50' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=seq(1..156250000)' ",
+               "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=100' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=seq(1..156250000)' ",
+               "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=150' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=seq(1..156250000)' "]
```

Comment:
What actually matters is not the rate threads, but the throttle.
You still need enough threads to reach that level, but 100 threads should be fine I think.

Reply:
Maybe 100 threads will be too much and 50 is better.

Reply:
done